docs/modules/reference/pages/configuration/performance-data-collection/introduction.adoc

```diff
@@ -0,0 +1,6 @@
+[[ref-performance-data-collectors]]
+= Collectors
+
+This section describes all available built-in performance data collectors.
+You can configure them to allow complex monitoring to meet the needs of your environment.
+For information on how to extend the functionality of these monitors, see the xref:development:overview/overview.adoc#overview[Development section] to contribute to the project.
```

Comment:
IMO, the development line looks better as a separate paragraph.  Make more of a distinction from just extending functionality by defining parameters or using metadata.
```suggestion

For information on how to extend the functionality of these monitors, see the xref:development:overview/overview.adoc#overview[Development section] to contribute to the project.
```